### DISCUSSION TITLE
v6.2.1

### DISCUSSION THREAD

**Discussion by snipe** [snipe/snipe-it#13691] [General]
## ⚠️ IMPORTANT: Future versions of Snipe-IT will require PHP 8.1 or greater

Oh hai there! v6.2.1 of Snipe-IT fixes a weird but important bug where if you had ANY encrypted custom fields, the asset history would show the changed values as encrypted, even if that specific field wasn't. We also did some polishing up of some of the UI elements, adding a copy icon next to license product keys and assets serials. (Let us know if you want that added elsewhere - those two seemed like a good start.) We also resolved a small usability bugaboo where when you'd click on the "new" icon, the name field (generally the first field in the modal) now takes the focus, which means you can start typing without having to click into the name field. 

We've also improved the UI of the "Integrations" setting, where you can no longer add a non-URL ("test", for example) and save those settings. For the full details, see the changelog below!

We're going to double-down on focusing on Snipe-IT v7, so expect things to be a little quiet for a bit and then very, very loud. Snipe-IT v7 will require at least PHP v8.1 (and will support PHP v8.2+) and will clear the path for cool stuff like customizable action hooks, custom fields on more than just assets, and a bunch of other neat features. 

We did have to comment out the new feature we were SO excited about - draggable columns. Nobody is sadder about that than we are, but in some circumstances, dragging a column would result in the data in that column being mismatched, so we had to make the very difficult call to pull the feature for now. We're still working on it, and we know how important a feature it is to many of you, but we have to make sure it's bulletproof this next time. 

## What's Changed
* Added more info in settings by @snipe in https://github.com/snipe/snipe-it/pull/13650
* Fixed bug where checkout to location would throw an error if FMCS was enabled by @snipe in https://github.com/snipe/snipe-it/pull/13655
* Fixed exception being thrown when selected import was deleted by @marcusmoore in https://github.com/snipe/snipe-it/pull/13638
* Commented out draggable columns by @snipe in https://github.com/snipe/snipe-it/pull/13657 🫠
* Fixed #13658 for asset history with encrypted fields by @snipe in https://github.com/snipe/snipe-it/pull/13667
* Fixed #13670 - order number missing from license import by @snipe in https://github.com/snipe/snipe-it/pull/13671
* Fixed #13662 - added clipboard.js by @snipe in https://github.com/snipe/snipe-it/pull/13672
* Fixed bug where license checkout/checkin notes were not being saved by @snipe in https://github.com/snipe/snipe-it/pull/13674
* Set modal focus to modal-name field by @snipe in https://github.com/snipe/snipe-it/pull/13677
* Fieldset Properly Sortable in Asset Models Table by @spencerrlongg in https://github.com/snipe/snipe-it/pull/13678
* Fixed General Webhooks error messaging and validation by @Godmartinz in https://github.com/snipe/snipe-it/pull/13679
* Fixed uncaught type error when attempting to render barcodes with invalid characters by @marcusmoore in https://github.com/snipe/snipe-it/pull/13681
* Fixed consumable user view route name by @snipe in https://github.com/snipe/snipe-it/pull/13683
* Updated language strings by @snipe in https://github.com/snipe/snipe-it/pull/13690

**Full Changelog**: https://github.com/snipe/snipe-it/compare/v6.2.0...v6.2.1

<hr /><em>This discussion was created from the release <a href='https://github.com/snipe/snipe-it/releases/tag/v6.2.1'>v6.2.1</a>.</em>

**Comment by bigolbug** [comment]
I love Snipe-IT. It is so feature rich, keep up the great work. I am currently on 6.2.0
I know this is slightly different from a copy feature but a clone button for both accessories and consumables would be awesome. 

Thank you again for all the hard work!

**Comment by Heckle0** [comment]
Been waiting on v7 any news???

**Reply by snipe** [reply]
Prolly Jan 2024. It's too close to the holidays (where half our team is on vacation) to try to launch it before the new year.

**Reply by snipe** [reply]
https://grokstar.dev/news/snipeit-releases/2023/11/snipe-it-v7-roadmap/

**Reply by Heckle0** [reply]
Thank you for that i somehow missed it. Cant wait to see the improvements.

**Reply by Heckle0** [reply]
I'm assuming things are running behind. Really looking forward to V.7. Any new news?
